docs/diagrams/AnnotationActivityDiagram.puml

```diff
@@ -0,0 +1,35 @@
+@startuml
+start
+:User executes annotate command;
+:Mark retrieves bookmark in query;
+:Mark retrieves cache copy in query;
+:Mark retrieves offline document of cache copy;
+
+'Since the beta syntax does not support placing the condition outside the
```

Comment:
do we need to join back the branching with merge nodes?

Reply:
my bad, thanks